peachjam_api/serializers.py

```diff
@@ -160,12 +176,15 @@ class JudgmentSerializer(BaseSerializerMixin, serializers.ModelSerializer):
     court = CourtSerializer(read_only=True)
     url = serializers.SerializerMethodField()
     judges = serializers.SerializerMethodField()
-    case_numbers = serializers.SerializerMethodField()
+    case_numbers = CaseNumbersSerializer(many=True, read_only=True)
```

Comment:
pls sanity check that these three related fields are loaded with prefetch_related / select_releated

Reply:
I've added the related lookups for case_numbers, judges and taxonomies